react-app/src/app/screens/Login/index.js

```diff
@@ -1,28 +1,20 @@
 import React, { Component } from 'react';
-import { Route, Redirect } from 'react-router-dom';
 import { connect } from 'react-redux';
-import PropTypes from 'prop-types';
 
 import style from './styles.scss';
 import LoginForm from './components/LoginForm';
 
-// TODO: delete the visualization of 'logged' commment
 class Login extends Component {
   render() {
     return (
       <div className={style.loginLayout}>
         <h2>Login</h2>
```

Comment:
This component is better, but there is still some work to do in order to apply the smart/dumb pattern. This component shouldn't be applying classnames or titles, that is layout job